packages/x-adapter/src/http-clients/__mocks__/fetch.mock.ts

```diff
@@ -19,14 +16,21 @@ export const koFetchMock = jest.fn(() =>
  *
  * @internal
  */
-export function fetchMock(response: any): any {
-  return (_url: string, { signal }: any) => {
-    return new Promise((resolve: AnyFunction, reject: AnyFunction) => {
+export function fetchMock(
+  response: unknown
```

Comment:
`unknown` is still type safe, while `any` is not. With `unknown` you prevent the developer to access/use any of the variable properties. With `any` you still allow them. There are only a few good `any` use cases, which essentially means "no types here", usually it can be replaced with `unknown` which is stricter.